plugins/CoreUpdater/Controller.php

```diff
@@ -95,7 +97,7 @@ public function getUpdaterJs()
 
         $files = array(
             "node_modules/jquery/dist/jquery.min.js",
-            "node_modules/jquery-ui-dist/jquery-ui.min.js",
+            "node_modules/jquery-ui-dist/jquery-ui.js",
```

Comment:
any reason for switching to the unminified version?

Reply:
I must have edited `getUpdaterJs()` instead of `getUpdaterCss()` at some point. :facepalm: I've reverted it.